xml/zypper_upgrade.xml

```diff
@@ -67,119 +67,149 @@
-     <step>
       <para>
-       Once done, check your repository configuration with:
+       All remaining repositories you want to keep need to be updated to the
+       new version. For this you need to remove the old version and add the
+       new one. This procedure is done in the following with an example
+       repository that has the following data:
       </para>
-<screen>&prompt.user;zypper lr -d</screen>
+      <simplelist>
+       <member>Name: <literal>M17N:fonts</literal></member>
+       <member>URI: <literal>https://download.opensuse.org/repositories/M17N:/fonts/openSUSE_Leap_15.2/</literal></member>
+      </simplelist>
+      <substeps>
+       <step>
+        <para>
+         Remove the old repository:
+        </para>
+        <screen>&prompt.sudo;zypper rr "M17N:fonts"</screen>
+       </step>
+       <step>
+        <para>
+         Add the new repository:
+        </para>
+        <screen>&prompt.sudo;zypper ar -f \
+        https://download.opensuse.org/repositories/M17N:/fonts/openSUSE_Leap_15.3/ \
+        "M17N:fonts"</screen>
+       </step>
+       <step>
+        <para>
+         Repeat this step for every repository that needs to be lifted to the
+         new version.
+        </para>
+        <para>
+         If adding a new repository fails, check in your browser whether the
+         URl is correct. If not, a new version of the repository may not
+         exist, yet. Deactivate the repository and try again later. Packages
+         from deactivated repositories will not be removed if all dependencies
+         are still fulfilled.
+        </para>
+        <screen>&prompt.sudo;zypper mr -d <replaceable>NAME</replaceable></screen>
+       </step>
+      </substeps>
      </step>
     </substeps>
    </step>
    <step>
     <para>
-     Refresh local metadata and repository contents with <command>zypper
-     ref</command>.
+     Finally, check the current repository configuration again
+    </para>
+    <screen>&prompt.sudo;zypper --releasever=15.3 lr -u</screen>
+    <para>
+     All repositories should now point to a <literal>15.3</literal>
+     version (except for 3rd party respositories without versioning that
+     support new releases). Repositories still pointing to an old version
+     should at least be disabled (the column <guimenu>Enabled</guimenu>
+     needs to show <literal>No</literal>).
     </para>
    </step>
    <step>
     <para>
-     Update Zypper and the package management itself with <command>zypper
-      patch --updatestack-only</command>.
+     Update the repository metadata by running:
     </para>
+    <screen>&prompt.sudo;zypper --releasever=15.3 ref</screen>
    </step>
    <step>
     <para>
-     Run the actual distribution upgrade with <command>zypper dup</command>.
-     You are asked to confirm the license of &productname; and of some
-     packages&mdash;depending on the set of installed packages.
+     Start the distribution upgrade that will lift your systemn to
```

Comment:
Typo fix:

```suggestion
     Start the distribution upgrade that will lift your system to
```

Also not a 100% sure about the word "lift" -- "upgrade" seems more understandable to me.